bam2fastq

```diff
@@ -2,45 +2,35 @@
 
 in_bam=$1 && shift
 
-picard_dir="/data/tools/picard-tools_v1.141"
-samtools_dir="/data/tools/samtools_v1.10"
+# versions
+echo "Version samtools:"
+samtools --version
 
 ## use 80 percent of available threads and memory
 threads=$( expr "$(grep -c processor /proc/cpuinfo)" / 10 \* 8)
+threads=3
```

Comment:
Try to avoid specifics like this as you want your container to be runnable on basically any host. I would either use that the thread defaulting magic or make it possible to pass it as a parameter